content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,90 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly classify ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
```

Comment:
Is there a link to this we can point to?  @myidpt  Also, is this only relevant when SDS is enabled?

Reply:
It should work regardless of SDS is enabled.

Reply:
nit: "You no longer need to" -> "Removes the need to"

Reply:
@rcaballeromx any thoughts?  I believe you came up with the original wording for this change.

Reply:
@incfly I have accepted your suggestion, thanks!